src/Workspaces/CSharp/Portable/CodeGeneration/EnumMemberGenerator.cs

```diff
@@ -123,9 +123,11 @@ private static ExpressionSyntax CreateEnumMemberValue(EnumDeclarationSyntax dest
                                 {
                                     // The user is left shifting ones, stick with that pattern
                                     var shiftValue = IntegerUtilities.LogBase2(value);
+
+                                    // Re-use the numericLiteral text so type suffixes match too
                                     return SyntaxFactory.BinaryExpression(
                                         SyntaxKind.LeftShiftExpression,
-                                        SyntaxFactory.LiteralExpression(SyntaxKind.NumericLiteralExpression, SyntaxFactory.Literal("1", 1)),
+                                        SyntaxFactory.LiteralExpression(SyntaxKind.NumericLiteralExpression, SyntaxFactory.Literal(numericLiteral.Token.Text, 1)),
```

Comment:
I couldn't see any way to find or specify the type suffix for these, but given the `ValueText == "1"` check above, this seemed entirely safe to me. LMK if not.